synapse/models/orgs.py

```diff
@@ -94,6 +100,32 @@ def getModelDefs(self):
                         'doc': 'Alias for the organization',
                     }),
                 )),
+                ('ou:orgnet4', {}, (
+                    ('org', ('ou:org', {}), {
+                        'ro': True,
+                        'doc': 'Org guid',
+                    }),
+                    ('net', ('inet:net4', {}), {
+                        'ro': True,
+                        'doc': 'Netblock owned by the organization',
+                    }),
+                    ('name', ('str', {'lower': True, 'strip': True}), {
+                        'doc': 'The name that the organization assigns to this netblock'
+                    }),
+                )),
+                ('ou:orgnet6', {}, (
```

Comment:
Quick / crazy brainstorm.  Since technically a net4 has a net6 equiv, could we maybe unify these some how?